build-system/tasks/prettify.js

```diff
@@ -46,7 +46,10 @@ const tempDir = tempy.directory();
  * Checks files for formatting (and optionally fixes them) with Prettier.
  */
 async function prettify() {
-  const filesToCheck = getFilesToCheck(prettifyGlobs, {dot: true});
+  // Prettier ignores the `.prettierignore` file when using the API like we
+  // are. So we need to filter our files down before feeding them to the API.
+  const ignore = fs.readFileSync('.prettierignore', 'utf8');
+  const filesToCheck = getFilesToCheck(prettifyGlobs, {dot: true, ignore});
```

Comment:
I think this should be passed in separately via a new param called `ignoreFile`. Directly adding an `ignore` field to `options` will pass it in to `globby.sync`, and I'm not sure the formats match. (This is a single string containing a block of files, which I'm not sure `globby.sync` knows to parse.)

Reply:
Ahh, fixing.